Sources/StreamChat/APIClient/Endpoints/EndpointPath.swift

```diff
@@ -0,0 +1,87 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+enum EndpointPath: Codable {
+    case connect
+    case sync
+    case users
+    case guest
+    case members
+    case search
+    case devices
+
+    case channels
+    case createChannel(String)
+    case updateChannel(String)
+    case deleteChannel(String)
+    case channelUpdate(String)
+    case muteChannel(Bool)
+    case showChannel(String, Bool)
+    case truncateChannel(String)
+    case markChannelRead(String)
+    case markAllChannelsRead
+    case channelEvent(String)
+    case stopWatchingChannel(String)
+    case pinnedMessages(String)
+    case uploadAttachment(String, String)
+
+    case sendMessage(ChannelId)
+    case message(MessageId)
+    case editMessage(MessageId)
+    case deleteMessage(MessageId)
+    case replies(MessageId)
+    case reactions(MessageId)
+    case addReaction(MessageId)
+    case deleteReaction(MessageId, MessageReactionType)
+    case messageAction(MessageId)
+
+    case banMember
+    case flagUser(Bool)
+    case flagMessage(Bool)
+    case muteUser(Bool)
+
+    var value: String {
```

Comment:
Do we have existing tests for all the cases listed here?

Reply:
Yeah, fortunately enough, the current XXXEndpoint_Tests were covering those, and I updated those.